src/main/java/uk/gov/justice/digital/delius/service/OffenderUpdatesService.java

```diff
@@ -23,9 +25,17 @@ public OffenderUpdatesService(OffenderDeltaService offenderDeltaService) {
     }
 
     public Optional<OffenderDelta> getNextUpdate() {
+        var maybeOffenderUpdate = retrieveNext(offenderDeltaService::lockNextUpdate); // Force a future date as we want ALL
+        if (maybeOffenderUpdate.isEmpty()) {
+            maybeOffenderUpdate = retrieveNext(() -> offenderDeltaService.lockNextFailedUpdate(LocalDateTime.now().minusMinutes(10)));
```

Comment:
I think we should add a info log message if a failed update i found so we can monitor occurrences of these exceptionally scenarios